scripts/apps/archive/views/related-items-preview.html

```diff
@@ -2,21 +2,17 @@
 
 <div ng-if="!loading" class="sd-list-item-group sd-list-item-group--space-between-items">
     <div class="sd-list-item sd-list-item--no-hover sd-shadow--z1"
-        ng-repeat="(key, relatedItem) in relatedItems track by key"> 
+        ng-repeat="(key, relatedItem) in relatedItems track by key">
         <div class="sd-list-item__column"><i class="type-icon" sd-filetype-icon data-item="relatedItem"></i></div>
         <div class="sd-list-item__column sd-list-item__column--grow sd-list-item__column--no-border">
-            <div class="sd-list-item__row">
-                <span class="sd-list-item__slugline" ng-if="relatedItem.slugline">{{ relatedItem.slugline }}</span>
-                <span class="sd-overflow-ellipsis sd-list-item--element-grow" sd-item-container data-item="relatedItem" class="container"></span>
-            </div>
-            <div class="sd-list-item__row">
-                <span class="state-label state-{{relatedItem.state}}">{{ relatedItem.state }}</span>
-                <span class="sd-overflow-ellipsis sd-list-item--element-grow" title="{{ relatedItem.headline }}">{{ relatedItem.headline || relatedItem.description_text }}</span>
-            </div>
+            <sd-related-item-in-list-react item="relatedItem" />
         </div>
+        <figure class="sd-list-item__thumbnail" ng-if="relatedItem.associations.featuremedia">
```

Comment:
Won't thumbnails be available in `sd-related-item-in-list-react`?

Reply:
No, `sd-related-item-in-list-react` is solely responsible for showing the first line and second line content as per the config.

Reply:
The same thumbnails snippet is duplicated in `scripts/apps/relations/views/related-items.html`

Could you change `sd-related-item-in-list-react` to include all the markup for [the entire item](https://user-images.githubusercontent.com/2076953/73247212-ab2b1b80-41b0-11ea-81fa-24189177d369.png)?

You could pass a `preview: boolean` or `editable: boolean` to determine whether to show actions